internal/pkg/errors/error_message.go

```diff
@@ -58,10 +58,10 @@ const (
 	InvalidCloudSuggestions            = "To list available connector plugin types, use `confluent connect plugin list`."
 	ConnectLogEventsNotEnabledErrorMsg = "Connect Log Events are not enabled for this organization"
 
-	// environment command
-	EnvNotFoundErrorMsg    = `environment "%s" not found`
-	EnvNotFoundSuggestions = "List available environments with `confluent environment list`."
-	EnvSwitchErrorMsg      = "failed to switch environment: failed to save config"
+	// environment & organization command
+	EnvNotFoundErrorMsg            = `environment "%s" not found`
+	OrgResourceNotFoundSuggestions = "List available %[1]ss with `confluent %[1]s list`."
```

Comment:
What does this syntax mean?

Reply:
`[n]` is an argument index, so `%[1]s` means 'first argument formatted as a string'

I use this to repeat the argument twice instead of having to write the argument twice.

Reply:
Didn't know you could do that, thanks!